spinnaker/resource_pipeline_template.go

```diff
@@ -0,0 +1,197 @@
+package spinnaker
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"reflect"
+
+	"github.com/armory-io/terraform-provider-spinnaker/spinnaker/api"
+	"github.com/ghodss/yaml"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourcePipelineTemplate() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"template": {
+				Type:             schema.TypeString,
+				Required:         true,
+				DiffSuppressFunc: suppressEquivalentPipelineTemplateDiffs,
+			},
+			"url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+		Create: resourcePipelineTemplateCreate,
+		Read:   resourcePipelineTemplateRead,
+		Update: resourcePipelineTemplateUpdate,
+		Delete: resourcePipelineTemplateDelete,
+		//Exists: resourcePipelineTemplateExists,
```

Comment:
Did you want to leave this commented out? I noticed `resourcePipelineTemplateExists` is implemented below.